modules/cli/src/main/scala/scala/cli/commands/DefaultBase.scala

```diff
@@ -3,28 +3,48 @@ package scala.cli.commands
 import caseapp._
 import caseapp.core.Error
 
+import scala.build.internal
+//import scala.cli.commands.SharedOptions
+
+case class DefaultOptions(
+  @Recurse
+  runOptions: RunOptions = RunOptions(),
+  version: Option[Boolean] = None
```

Comment:
FYI, I we want to support `-version` (single dash), I think the following would work
```suggestion
  @Name("-version")
    version: Option[Boolean] = None
```

Reply:
Why an `Option[Boolean]`? A simple `Boolean` should do (defaulting to `false`).